src/components/DatePicker/transform-value-to-date-or-range.ts

```diff
@@ -38,17 +37,34 @@ export const transformRelativeDateObjectToDate = (
 };
 
 /*
-Read a `RelativeDate` and find the corresponding `Date` relative to the moment of execution.
+Read a `RelativeDate` and find the corresponding `Date` relative to the variable passed as `date` property.
 */
-export const transformRelativeDateToDate = (relativeDate: RelativeDate): Date | undefined => {
-  // In relative date we always use today as date to update
-  const today = new Date(Date.now());
-  const date = DateTime.fromJSDate(today, { zone: 'UTC' }).toJSDate();
-  // retrieve date from luxon
-  return transformRelativeDateObjectToDate({
-    ...relativeDate,
-    date,
+export const transformRelativeDateToDate = (
+  relativeDateRange: RelativeDateRange,
+  relativeAvailableVariables: VariablesBucket = [],
+  variableDelimiters: VariableDelimiters = { start: '', end: '' },
+): Date | undefined => {
+  // start is always a date as relativeAvailableVariables are not customizable
```

Comment:
Not sure I got this comment on first read.
Could you confirm if this is correct : relativeAvailableVariables always contains Dates, and they can be changed by a user, in the UI.

Reply:
I will had a proper explanation: it not relevant since calculated date can be in the past or the future now.
Before we always have start date as reference for start so it was always the selected variable and this variable comes from the relative available variables which is an array that conceptor cannot customize ( unless availableVariables that are customizable, but we don't really care about this information here)